### DISCUSSION TITLE
How do i set the image size in markdown while keeping next/image working?

### DISCUSSION THREAD

**Discussion by DvdPlaat** [shuding/nextra#2726] [Q&A]
Hi i want to include images with a specific size but i was wondering how i could do this using the `![]()` syntax, using mdx is not really a option atm

**Comment by dimaMachina** [comment]
there is no builtin way, possible solutions https://dzone.com/articles/how-to-style-images-with-markdown

**Reply by DvdPlaat** [reply]
Was hoping for there to be a extension around markdown for it somehow but i guess ill just have to make mdx work thanks.
